src/main/java/com/rtb/projectmanagementtool/task/TaskTreeData.java

```diff
@@ -0,0 +1,64 @@
+package com.rtb.projectmanagementtool.task;
+
+import java.util.ArrayList;
+
+/** Class containing task data. */
+public final class TaskTreeData {
```

Comment:
nit:  if we're using "Data" as a suffix for database data models, we should avoid that name here (since the tree itself is not stored in the database).